supporter-product-data/cloudformation/dynamo-tables.yaml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Supporter product data tables - stack name should be supporter-product-data-tables-${Stage}
+Parameters:
+  Stage:
+    Description: Environment name ( CODE, UAT or PROD)
+    Type: String
+    AllowedValues:
+      - PROD
+      - UAT
+      - CODE
+    Default: CODE
+Mappings:
+  StageVariables:
+    CODE:
+      DynamoWriteCapacity: 1
+      DynamoReadCapacity: 5
```

Comment:
why do we need 5? also did you consider on-demand instead for dev/uat databases?

Reply:
I didn't actually though that's not a bad idea, I'll look into it

Reply:
Ok I've updated this to use on demand for code/uat